src/index.js

```diff
@@ -44,12 +52,82 @@ const updateLocal = () => { // add to Local storage
 };
 /* eslint-disable */
 const completeTask = (index) => { // function for change the comple status to complete
-  tasks[index].completed = !tasks[index].completed; //
+  tasks[index].completed = !tasks[index].completed; //change the status
+  if (tasks[index].completed) {
+    todoItemElems[index].classList.add('checked');
+  } else {
+    todoItemElems[index].classList.remove('checked');
+  }
+  updateLocal();
+  addToHTML();
 };
 /* eslint-anable */
 addTasksBtn.addEventListener('click', () => { // by clicking the button
   tasks.push(new Task(deskTaskInput.value)); // add new task
   updateLocal(); // save to Local Storage
   addToHTML(); // call function wich add new task to HTML template
   deskTaskInput.value = ''; // clear the input after clicking "Add" button
-});
\ No newline at end of file
+
+});
+
+const deleteTask = index => { // function for delete button
+  todoItemElems[index].classList.add('deletion');
+  setTimeout(() => { // create a pause for animation when delete
+    tasks.splice(index, 1); // splice delete the index
+    updateLocal();
+    addToHTML();
+  }, 500);
+};
+
+// const editTask = index => {
+//   let span = tasks[index].description;
+//   // todoItemElems[index].classList.add('edition');
+//   let aaa = document.createElement('input');
+//   aaa.type = 'text';  
+//   span = aaa.value;
+// };
+
+const draggables = document.querySelectorAll('.draggable')
+const containers = document.querySelectorAll('.todos-wrapper')
+
+draggables.forEach(draggable => {
+  draggable.addEventListener('dragstart', () => {
+    draggable.classList.add('dragging')
+  })
+
+  draggable.addEventListener('dragend', () => {
+    draggable.classList.remove('dragging')
+  })
+})
+
+containers.forEach(container => {
+  container.addEventListener('dragover', e => {
+    e.preventDefault()
+    const afterElement = getDragAfterElement(container, e.clientY)
+    const draggable = document.querySelector('.dragging')
+    if (afterElement == null) {
+      container.appendChild(draggable)
+    } else {
+      container.insertBefore(draggable, afterElement)
+    }
+  })
+})
+
+function getDragAfterElement(container, y) {
+  const draggableElements = [...container.querySelectorAll('.draggable:not(.dragging)')]
+
+  return draggableElements.reduce((closest, child) => {
+    const box = child.getBoundingClientRect()
+    const offset = y - box.top - box.height / 2
+    if (offset < 0 && offset > closest.offset) {
+      return {
+        offset: offset,
+        element: child
+      }
+    } else {
+      return closest
+    }
+  }, {
+    offset: Number.NEGATIVE_INFINITY
+  }).element
```

Comment:
- [ ] From the project specifications listed [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md), you were supposed to create two different files  ⬇️ 
![image](https://user-images.githubusercontent.com/33067256/127524522-530435f9-ab3c-4d21-a387-db4549418d85.png)

- [ ] When i drag an item to another index, it should update the items index as stated here ⬇️ 
![image](https://user-images.githubusercontent.com/33067256/127525051-5f8ac7de-c0f7-43de-9df9-c679dc55e643.png)
```
Example
I have three todo's with different indexes, A,B,C. If i move C to replace B, C's new index should be B's previous index. 
```
Hope that is clear enough

- [ ] When i click on completed on a task, it should change from `false` to `true` and vice versa


- [ ] From the Learning Objectives listed [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md) the last point states that you should use ES6 modules to write modular js. This means using `arrow functions` over the `function` keywork. Kindly fix this also